hil-test/tests/embassy_interrupt_executor.rs

```diff
@@ -3,6 +3,12 @@
 
 //% CHIPS: esp32 esp32c2 esp32c3 esp32c6 esp32h2 esp32s2 esp32s3
 //% FEATURES: unstable embassy
+//% ENV(single_integrated):   ESP_HAL_EMBASSY_CONFIG_TIMER_QUEUE = single-integrated
```

Comment:
Could we get some docs somewhere on how to use this? :pleading_face:  I think I understand now, but it wasn't immediately clear. I think a new contributor would also struggle to grok this.

Reply:
Or maybe less how to use it, but more what this does to the tests we're about to run

Reply:
then probably it's worth to add documentation about the meta-keys

Reply:
I can do that but I don't understand TAG I think.

Reply:
TAG is just a name and only used to sort examples for `run-example` w/o specifying examples (i.e. run all examples one after each other) - but we can leave that out and I document that one (since I added it 😄 )

Reply:
Added docs to xtask/README.md, let me know if they make sense.